src/util/battery/battery.cpp

```diff
@@ -31,12 +31,12 @@ Battery::~Battery() {
 
 Battery* Battery::getBattery(QObject* parent) {
 #ifdef __BATTERY__
-#ifdef Q_OS_LINUX
-    return new BatteryLinux(parent);
-#elif defined(Q_OS_WIN)
+#if defined(Q_OS_WIN)
     return new BatteryWindows(parent);
 #elif defined(Q_OS_MAC)
     return new BatteryMac(parent);
+#else
+    return new BatteryLinux(parent);
```

Comment:
I made `BatteryLinux` the default here because otherwise we'd have to enumerate 
`Q_OS_LINUX`, `Q_OS_FREEBSD`, `Q_OS_OPENBSD` and so on..

Reply:
I agree with this pragmatic approach. If this should ever fail in the future we could add a more fine-grained distinction for the "other" OSes.